change_log/next/FE-1876-help-component.yml

```diff
@@ -0,0 +1 @@
+Bug Fixes: "Fix issue with outline border not being visible when component is focused"
```

Comment:
Could you add that it was the `Help` component that was fixed as we need to be explicit for release notes